database/migrations/2023_10_31_204320_create_wikibase_notification_sent_records_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateWikibaseNotificationSentRecordsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('wikibase_notification_sent_records', function (Blueprint $table) {
+            $table->id();
+            $table->timestamps();
+
+            $table->integer('wiki_id')->nullable()->unsigned();
+
+            $table->string('notification_type') -> nullable();
```

Comment:
I don't think the `notification_type` should be nullable. It's the most important piece of information in such a row, so we should enforce that it's being set.